Kconfig

```diff
@@ -940,6 +940,11 @@ menu "LVGL configuration"
 		config LV_USE_LIST
 			bool "List"
 			default y if !LV_CONF_MINIMAL
+		config LV_USE_LOTTIE
+			bool "Lottie"
+			default n
+			help
```

Comment:
depends on LV_USE_VECTOR_GRAPHIC && (LV_USE_THORVG_INTERNAL || LV_USE_THORVG_EXTERNAL)

Reply:
I intentionally didn't add it to make this feature easier to discover. With this to make Lottie appear a some other things needs to be enabled.

Anyway, I'm not an active Kconfig user. If you say it's better like this, I'm ok with that.

Reply:
After adding this, you can remind the user to open the configuration item during the menuconfig process, without waiting for the compilation error to check again, reducing the occurrence of secondary configuration check